README.md

```diff
@@ -7,7 +7,7 @@ Intel® Media SDK provides an API to access hardware-accelerated video decode, e
 
 # Important note
 The current version of Intel Media SDK is only compatible with the Linux* Graphics Driver from [Intel® Media Server Studio for Linux*](https://software.intel.com/en-us/intel-media-server-studio). You need to download and install Intel Media Server Studio 2017 R2 _before_ you build Intel Media SDK.
-Intel Media SDK depends on a special version of [LibVA](https://www.freedesktop.org/wiki/Software/vaapi/) which comes with Intel Media Server Studio installation and is not in upstream, so this version is not compatible with the LibVA/driver available at [01org](https://01.org/linuxgraphics/downloads ).
+Intel Media SDK depends on a special version of [LibVA](https://www.freedesktop.org/wiki/Software/vaapi/) which comes with Intel Media Server Studio installation and is not in upstream, so this version is not compatible with the LibVA/driver available at [01org](https://01.org/linuxgraphics/downloads). Installed libav-devel,libdrm-devel/libX11-devel should be sufficient for build environment to compile.
```

Comment:
Two important moments:
 - Cmake handles dependencies already.
 - Using mentioned packages different from what comes with Intel MSS installation may lead to broken functionality of Media SDK, so I can't agree with this addition.